### ISSUE TITLE
[Error] TypeError: null is not an object (evaluating 'e.style') 	d (ionic.scroll.sista.min.js, line 1) 	b (ionic.scroll.sista.min.js, line 1) 	(anonymous function) (ionic.scroll.sista.min.js, line 1)

### ISSUE THREAD

**Issue by mabdelfattah** [djett41/ionic-scroll-sista#6]
The plugin doesn't work and I'm getting the following error in the console. I'm using latest Ionic (1.1) and your plugin latest version.

```
[Error] TypeError: null is not an object (evaluating 'e.style')
        d (ionic.scroll.sista.min.js, line 1)
        b (ionic.scroll.sista.min.js, line 1)
        (anonymous function) (ionic.scroll.sista.min.js, line 1)
```

**Comment by djett41** [comment]
I just released a new version which I believe should also resolve this issue.  Try 1.0.4 thanks!!

**Comment by djett41** [comment]
Closing this issue as it should be fixed in new release

**Comment by mabdelfattah** [comment]
It didn't fix! I'm getting the same error!

**Comment by djett41** [comment]
@mabdelfattah Can you post your code on a codepen? Or see if you can reproduce this somehow on the demo?  If I can reproduce it I should be able to fix it.  Thanks!

**Comment by mabdelfattah** [comment]
it is the same :)
[Error] TypeError: null is not an object (evaluating 'e.style')
    f (ionic.scroll.sista.min.js, line 1)
    S (ionic.scroll.sista.min.js, line 1)
    (anonymous function) (ionic.scroll.sista.min.js, line 1)

**Comment by djett41** [comment]
Ok thanks, but can you post the code where you are using the directive?  I want to replicate the issue so I can start trying to fix it.  Thanks

**Comment by djett41** [comment]
I think this is fixed as a result of upgrade to new 1.0.5 version. Install the latest from bower and try again @mabdelfattah Thanks!
